Controllers/Project/UserController.cs

```diff
@@ -87,22 +87,14 @@ public IActionResult UpdateUser(int userId, UpdateRequest editUser)
             return BadRequest();
         }
 
-        // This is redundant code
-        // if (HttpContext.User == null)
-        // {
-        //     return Unauthorized("Unable to find user, returns null");
-        // }
+        _userRepository.UpdateUser(claimId, editUser);
+        // return Ok(new { message = "User updated" });
+
```

Comment:
This is not needed because userId is a parameter and claimId is passed in as the parameter value (see below).  It's impossible for anyone but the current user to edit.  In addition, there is no route available to even change the ID manually.  Users can only update with this route https://localhost:5001/api/User/current/edit 

_userRepository.UpdateUser(claimId, editUser);
  var user = _userRepository.GetUserById(claimId);